src/player.js

```diff
@@ -433,7 +433,8 @@ export default class Player extends FakeEventTarget {
     this._setConfigLogLevel(config);
     this._playerId = Utils.Generator.uniqueId(5);
     this._prepareVideoElement();
-    const playsInline = Utils.Object.getPropertyPath(config, 'playback.playsinline');
+    const defaultPlaysInline = Utils.Object.getPropertyPath(Player._defaultConfig, 'playback.playsinline');
```

Comment:
if we have an issue with our default we have to fix it for all default values not for `playsinline` only